roles/openshift_logging_elasticsearch/tasks/main.yaml

```diff
@@ -100,13 +100,22 @@
     files:
     - "{{ tempdir }}/logging-elasticsearch-view-role.yaml"
     delete_after: true
-
 # configmap
+- fail:
+    msg: "The openshift_logging_es_log_appenders '{{openshift_logging_es_log_appenders}}' has an unrecognized option and only supports the following as a list: {{es_log_appenders | join(', ')}}"
+  when:
+    - es_logging_contents is undefined
+    - "{{ openshift_logging_es_log_appenders | list | difference(es_log_appenders) | length != 0 }}"
```

Comment:
Recalled why we choose this approach.  The desire is to restrict them to the options of file and console.